docs/contributing-testing.rst

```diff
@@ -189,3 +189,24 @@ environment object.
 3. Delete all of the requirements lockfiles for the chosen environment, then regenerate them:
    ``for h in `riot list --hash-only "^${VENV_NAME}$"`; do rm .riot/requirements/${h}.txt; done; scripts/compile-and-prune-test-requirements``
 4. Commit the resulting changes to the ``.riot`` directory, and open a pull request against the trunk branch.
+
+What do I do when my pull request has failing tests unrelated to my changes?
+----------------------------------------------------------------------------
+
+The test suite is not completely reliable. There are usually some tests that can fail without any of their code paths being
+changed. This slows down development because most tests are required to pass for pull requests to be merged.
+
+The ``tests/utils`` module provides the ``@flaky`` decorator to enable contributors to handle this situation. As a contributor,
```

Comment:
nit - Could we link to the implementation?
```suggestion
The ``tests/utils`` module provides the [``@flaky`` decorator](https://github.com/DataDog/dd-trace-py/blob/623f2df4de802563a463acc4d3c000dbc742e3d3/tests/utils.py#L1285) to enable contributors to handle this situation. As a contributor,
```

Reply:
@emmettbutler I just previewed the change and it looks strange... Not sure how to make it a normal hyperlink